lib/compat/wordpress-6.9/template-activate.php

```diff
@@ -1,5 +1,52 @@
 <?php
 
+// Migrate existing "edited" templates. By existing, it means that the template
+// is active.
+function gutenberg_migrate_existing_templates() {
```

Comment:
This function retained its name but changed behaviour: it no longer sets the option, just computes the value for it.

Reply:
Fixed